README.md

```diff
@@ -98,106 +35,300 @@ Class.
+>
+  <div class="flex-shrink-0">
+    <img class="h-12 w-12" src="/asset/sanc.png" alt="Sanctuary Logo" />
+  </div>
+  <div>
+    <div class="text-xl font-medium text-black">Sanctuary Chat Ap</div>
+    <p class="text-gray-500">You have a new message!</p>
+  </div>
 </div>
 ```
 
-```css
-#homepage {
-  // Nothing in this file is outside of this wrapper!
-}
-```
+**Result:**
 
-### Use a SCSS Grid that can handle breakpoints from HTML only.
+<br>
 
-It's important to bake responsive layouts into HTML.  Rearranging a
-Layout for mobile is painstaking work, and it's even harder from CSS.
+![alt text](chat.png "Title")
 
-**Good:**
-```html
-<div class='small-12 medium-6 large-3 small-only-text-center hide-for-xlarge-up columns'></div>
-```
+<br>
+
+Tailwind's intuitive naming conventions allow us to visualize how this component will appear in our HTML. To remove or add an attribute, we just have to append or remove a class name. For example, if we wanted to change the background colour and give the sanctuary logo a border radius, we would append the following class names: `bg-darkest-grey` and `rounded-full` to our previously built ChatAppNotifcation component.
+
+<br>
+
+### Code:
+
+<br>
 
-**Bad:**
 ```html
-<div class='12-columns other-styling-done-from-css'></div>
+<div
+  class="p-6 max-w-sm mx-auto bg-darkest-grey rounded-xl shadow-md flex items-center space-x-4"
+>
+  <div class="flex-shrink-0 rounded-full">
+    <img class="h-12 w-12" src="/asset/sanc.png" alt="ChitChat Logo" />
+    .....
+  </div>
+</div>
 ```
 
-Some nice grids are:
-- http://foundation.zurb.com/grid.html
-- http://www.basscss.com/
+### Result:
+
+<br>
+
+![alt text](chat1.png "Title")
 
-### Frameworks *aren't* bad!
+This approach enables us to implement a completely custom component design without writing verbose and non-reusable class names. Let's try writing the exact `ChatAppNotifcation` component using semantic CSS
 
-CSS Frameworks get a bad rap.  Why use a big, bloated CSS framework
-when I can roll my own ninja-shit?  Here's some good reasons:
+```html
+<!-- ChatAppNotifcation.tsx -->
+
+<div class="chat-notification">
+  <div class="chat-notification-logo-wrapper">
+    <img
+      class="chat-notification-logo"
+      src="asset/sanc.png"
+      alt="ChitChat Logo"
+    />
+  </div>
+  <div class="chat-notification-content">
+    <h4 class="chat-notification-title">Sanctuary Chat App</h4>
+    <p class="chat-notification-message">You have a new message!</p>
+  </div>
+</div>
 
-- Grid Frameworks give you a battle-tested way of handling Responsive layouts
-- There's often Cross Browser CSS problems that Frameworks have already solved
-- It's (basically) a myth that a CSS framework will slow down the browser.
-- CSS Frameworks are designed to reduce complexity, making maintenance (way) easier!
-- You can always "not" use the grid for pages that don't make sense gridded!
+<style>
+  .chat-notification {
```

Comment:
Not strongly opinionated either way, but let's decide between `CamelCase` & `kebab-case`.

Personally I like CamelCase for object style, because it shows a clear difference between tailwind classes, and our internal classes